### PR TITLE
Add milcoff author

### PR DESCRIPTION
**Pull request by nmilcoff**
If you are submitting a new blog please read:

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines:

- I have a valid blog & RSS URL
- Host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- Blog is active with at least 3 Xamarin related blog posts in the last 6 months
- If the blog has mixed content (Xamarin and Personal/Non-Xamarin blogs) a filter has been applied
- If you delete your blog you will come delete your blog from Planet Xamarin
- Your blog may be removed at any time if any of these are broken.

### CODE REVIEW

File: src/Firehose.Web/Authors/NicolasMilcoff.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        public IEnumerable<Uri> FeedUris
+        {
+            get { yield return new Uri("http://nmilcoff.com/feed/"); }
+        }
+
+        public string TwitterHandle => "nmilcoff";
+
+        public string GravatarHash => "a689293f9c38aecdf729583533665e4a";
+
+        public string GitHubHandle => nmilcoff;
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/NicolasMilcoff.cs]: This should be a string

File: src/Firehose.Web/Authors/NicolasMilcoff.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        public string FirstName => "Nicolas";
+
+        public string LastName => "Milcoff";
+
+        public string StateOrRegion => "Rosario, Argentina";
+
+        public string EmailAddress => "nicolas.milcoff@d-genix.com";
+
+        public string ShortBioOrTagLine => Mobile lead developer;
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/NicolasMilcoff.cs]: This should be a string

**Comment by Cheesebaron** [comment]
> Blog is active with at least 3 Xamarin related blog posts in the last 6 months

This is not fulfilled.

**Comment by nmilcoff** [comment]
>> Blog is active with at least 3 Xamarin related blog posts in the last 6 months

> This is not fulfilled.

True, didn't read that carefully enough. I'll close this PR and open a new one in a few days.

**Comment by Cheesebaron** [comment]
We can just reopen when you are ready

**Pull request closed without merging**